instrumentation/kafka-clients/src/main/java/brave/kafka/clients/KafkaTracing.java

```diff
@@ -243,4 +252,35 @@ static void addTags(ConsumerRecord<?, ?> record, SpanCustomizer result) {
     }
     result.tag(KafkaTags.KAFKA_TOPIC_TAG, record.topic());
   }
+
+
+  /**
+   * Avoids array allocation when logging a parameterized message when fine level is disabled. The
+   * second parameter is optional.
+   *
+   * <p>Ex.
+   * <pre>{@code
+   * try {
+   *    return message.getStringProperty(name);
```

Comment:
update this comment as it drifted both in the content here and the api used from logging (JUL doesn't have parameterizations)

Reply:
I don't quite follow? I copied this from `JmsTracing`, it's invoking the same API and I assume the parameterization is working (if not then it must be broken in `JmsTracing` as well).

Reply:
sorry I mean "message.getStringProperty" is not relevant in kafka as the name is not message and there's no method getStringProperty. basically the javadoc needs a fixup.

similarly the catch in the javadoc here has drifted.

make sense?

Reply:
I see - I've made the javadoc more generic (I don't think it makes sense to use an extremely specific example here as this method is general-purpose?)

Reply:
I don't even mind no javadoc, just don't want it wrong :P